scripts.js

```diff
@@ -0,0 +1,16 @@
+var text = 'Velociraptor is a genus of herbivorous ceratopsid dinosaur that first appeared during the late Maastrichtian stage of the late Cretaceous period.';
+
+var dinosaur = 'triceratops';
+var upperCasedDinosaur = dinosaur.toUpperCase();
+
+console.log(upperCasedDinosaur);
+
+var textAfter = text.replace ('Velociraptor', dinosaur);
+
+console.log(textAfter);
+
+console.log(textAfter.length/2);
+
+var textSliced = text.slice(0, 73);
```

Comment:
from where you take number 73? your code should be flexible so you need to have this value calculated in variable ;)

Reply:
I know ;) It should be 72, but then the result will looks like this:
"TRICERATOPS is a genus of herbivorous ceratopsid dinosaur that first ap"
Author wants:
"TRICERATOPS is a genus of herbivorous ceratopsid dinosaur that first app"